docs/src/troubleshooting.md

```diff
@@ -139,3 +139,8 @@ begin
 end
 ```
 
+
+## Distribution of cells on multiple MPI ranks in [`P4estMesh`](@ref)
```

Comment:
```suggestion
## MPI ranks are assigned zero cells in [`P4estMesh`](@ref) even though there are enough cells
```
We usually use a "problem statement" as the heading for in the troubleshooting section.